types/static-kdtree/static-kdtree-tests.ts

```diff
@@ -0,0 +1,58 @@
+import createKDTree = require("static-kdtree");
+
+// Create a bunch of points
+const points2D = [
+    [0, 1],
+    [-5, 0.11],
+    [0, 10],
+
+    // ...
+
+    [4, 3],
+] as Array<[number, number]>;
```

Comment:
your test pass without types assertions, just by specifying a type. Do you need types assertions for some reason?

Reply:
Nope. It would pass without extra specific typing. Leaving the assertion for this test case was to (hopefully) advocate a pattern in context of using this package. :)

Reply:
I meant:
```ts
const points2D: Array<[number, number]> = [
   ...
];
```
I'm pretty sure that would work, sorry for confusion

Reply:
Surely. That works too.